docs/Oracle Jobs/jobs.md

```diff
@@ -0,0 +1,272 @@
+observationTimeout = "10s"
+blockchainTimeout  = "20s"
+contractConfigTrackerSubscribeInterval = "2m"
+contractConfigTrackerPollInterval = "1m"
+contractConfigConfirmations = 3
+observationSource = """
+    // data source 1
+    ds1          [type=bridge name=voter_turnout];
+    ds1_parse    [type=jsonparse path="one,two"];
+    ds1_multiply [type=multiply times=1.23];
+
+    // data source 2
+    ds2          [type=http method=GET url="https://chain.link/voter_turnout/USA-2020" requestData="{\\"hi\\": \\"hello\\"}"];
+    ds2_parse    [type=jsonparse path="three,four"];
+    ds2_multiply [type=multiply times=4.56];
+
+    ds1 -> ds1_parse -> ds1_multiply -> answer1;
+    ds2 -> ds2_parse -> ds2_multiply -> answer1;
+
+    answer1 [type=median                      index=0];
+    answer2 [type=bridge name=election_winner index=1];
+"""
+```
+
+- `p2pPeerId`: the base58-encoded libp2p public key of this node.
+- `p2pBootstrapPeers`: a list of libp2p dial addresses of the other bootstrap nodes helping oracle nodes find one another on the network.
+- `keyBundleID`: the hash of the OCR key bundle to be used by this node (the Chainlink node keystore manages these key bundles — use the subcommands under `chainlink keys ocr` to interact with them or use the Operator UI).
+- `monitoringEndpoint`: the URL of the telemetry endpoint to send OCR metrics to.
+- `transmitterAddress`: the Ethereum address from which to send aggregated submissions to the OCR contract.
+- `observationTimeout`: the maximum duration to wait before an off-chain request for data is considered to be failed/unfulfillable.
+- `blockchainTimeout`: the maximum duration to wait before an on-chain request for data is considered to be failed/unfulfillable.
+- `contractConfigTrackerSubscribeInterval`: the interval at which to retry subscribing to on-chain config changes if a subscription has not yet successfully been made.
+- `contractConfigTrackerPollInterval`: the interval at which to proactively poll the on-chain config for changes.
+- `contractConfigConfirmations`: the number of blocks to wait after an on-chain config change before considering it worthy of acting upon.
+
+### VRF
+
+```toml
+uuid = "123e4567-e89b-12d3-a456-426655440000"
+type = "randomnesslog"
+schemaVersion = 1
+name = "vrf-primary"
+coordinatorAddress = "0xaba5edc1a551e55b1a570c0e1f1055e5be11eca7"
+confirmations = 6
+publicKey = "0x79BE667EF9DCBBAC55A06295CE870B07029BFCDB2DCE28D959F2815B16F8179800"
+observationSource="""
+getrandomvalue [type=vrf];
+"""
+```
+
+- `uuid`: the globally-unique ID of the job. User-specified or randomly-generated.
+- `coordinatorAddress`: the address of the VRFCoordinator contract coordinating requests and responses for randomness.
+- `confirmations`: the number of blocks to wait before a request is considered final enough to be worth responding to.
+- `publicKey`: the public key of the VRF keypair securing the randomness response.
+
+---
+
+## Writing pipelines
```

Comment:
Does this section fit here, or could we make it its own page?

Reply:
IMO it can go here for now. We will be fleshing this out more in future.